code-challenges.test.js

```diff
@@ -15,46 +17,152 @@
+const input = (num) => {
+    let array = [1,1];
+    for (let index = 2; index < num; index++) {
+        array.push(array[index-1] + array[index-2]);
+    }
+    return array
+}
 
+// //Take input of variable that includes an example number
+// //Establish an array with a length equal to input variable
+// //Create a For loop that adds two prior index values to current index location (fibonnaci sequence)
+// //Return array of numbers that gives the output of the fibonnaci sequence
 
+//Test PASS
 
-// --------------------2) Create a function that takes in an array and returns a new array of only odd numbers sorted from least to greatest.
+//Notes on process: Creating anonymous functions that coincide with jest tests has proven to be a really heavy learning hurtle for me. 
+// I knew what I had to do in order to make this function work, however, I kept overcomplicating it by adding in additional parameters.
+// The final function came with some help from Rebecca and she really helped simplify the approach to this.
 
-// a) Create a test with expect statements for each of the variables provided.
 
-var fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
-// Expected output: [-9, 7, 9, 199]
 
-var fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
-// Expected output: [-823, 7, 23]
+// // --------------------2) Create a function that takes in an array and returns a new array of only odd numbers sorted from least to greatest.
+
+// // a) Create a test with expect statements for each of the variables provided.
+
+describe("Accumulating Sum", () => {
+
+//     // a test/it method, nested within the describe block, that in plain words, describes that the function does.
+    it("Returns array with only numbers in numerical order", () => {
+  
+      //an expect method, nested within the test block, calling on the hello() function, followed by the .toEqual() matcher that checks the expected output of the function return.
+      expect(sorted([4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"])).toEqual(([-9, 7, 9, 199]))
+      expect(sorted(["hello", 7, 23, -823, false, 78, null, "67", 6, "number"])).toEqual(([-823, 7, 23]))
+    })
+  })
+
+//Test FAIL. ReferenceError: input is not defined
+
+// var fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
+// // Expected output: [-9, 7, 9, 199]
+
+// var fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
+// // Expected output: [-823, 7, 23]
 
 
 // b) Create the function that makes the test pass.
 
+//Take input of given array into a const
+//Create a filter that removes all types of data except for numbers
+//Order the filtered numbers from least to greatest
+//Return new array with only numbers
+
+const sorted = (val) => {
+    if(typeOf(val) === 'number'){
+        let newArray = sorted.slice().sort((a,b)=>a-b)
+            return newArray
+    
+    }
+    }
+    
+//Test FAIL. Received undefined / typeOf is not defined
```

Comment:
yea if you look up typeof you'll find its **_super weird_**. typeof gets an argument by take a "space"

```javascript
if(typeOf val === 'number')
```
             ^